### PR TITLE
K8SPSMDB-237 Add test cases

### PR DESCRIPTION
**Pull request by nonemax**
[K8SPSMDB-237](https://jira.percona.com/browse/K8SPSMDB-237)

### CODE REVIEW

File: e2e-tests/users/run [hunk 1]
```diff
@@ -31,7 +38,11 @@ check_mongo_auth "$user:$newpass@$cluster-0.$cluster.$namespace"
 check_mongo_auth "$user:$newpass@$cluster-1.$cluster.$namespace"
 check_mongo_auth "$user:$newpass@$cluster-2.$cluster.$namespace"
 
-desc 'test new MONGODB_BACKUP_USER'
+desc 'Run backup'
+run_backup aws-s3
+wait_backup "$backup_name_aws"
```
Comment (delgod) [inline e2e-tests/users/run:43]: let's run a backup after MONGODB_BACKUP_USER

Comment (nonemax) [reply e2e-tests/users/run]: done https://github.com/percona/percona-server-mongodb-operator/pull/407/commits/64f1b9a30df6ceb21d3ecaf378f0e5a991d339e4

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| init-deploy | passed |
| one-pod | passed |
| upgrade | passed |
| monitoring | passed |
| self-healing | failed |
| limits | passed |
| operator-self-healing | passed |
| upgrade-consistency | passed |
| scaling | passed |
| demand-backup | passed |
| monitoring-2-0 | failed |
| arbiter | passed |
| security-context | passed |
| scheduled-backup | passed |
| service-per-pod | passed |
| liveness | passed |
| users | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| one-pod | passed |
| storage | passed |
| init-deploy | passed |
| upgrade | passed |
| monitoring | passed |
| self-healing | passed |
| limits | passed |
| upgrade-consistency | passed |
| scaling | passed |
| monitoring-2-0 | failed |
| demand-backup | passed |
| security-context | passed |
| arbiter | failed |
| operator-self-healing | passed |
| scheduled-backup | failed |
| service-per-pod | passed |
| liveness | passed |
| users | passed |

**Pull request merged**